client/src/app/pages/pull-request-details/pull-request-details.component.html

```diff
@@ -1,21 +1,36 @@
-<div class="my-3">
+<div class="mt-4 mb-5">
   @if (query.data(); as pr) {
     <div class="flex flex-col items-start">
-      <span class="text-gray-500 text-sm uppercase font-semibold tracking-wider mb-1">Pull Request #{{ pr.number }}</span>
-      <div class="flex items-center gap-4">
+      <div class="flex items-center gap-4 mt-2">
```

Comment:
nitpick: for me this gap seems oddly big compared to the subheading

![image](https://github.com/user-attachments/assets/9bcb7f88-7c93-4b91-a08e-400a2a4a325c)